modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/topology/GridDhtLocalPartition.java

```diff
@@ -967,8 +978,13 @@ public long fullSize() {
      * @return Number of rows cleared from page memory.
      * @throws NodeStoppingException If node stopping.
      */
-    protected long clearAll(EvictionContext evictionCtx) throws NodeStoppingException {
-        long order = clearVer;
+    protected long clearAll(EvictionContext evictionCtx, PartitionsEvictManager.EvictReason reason) throws NodeStoppingException {
```

Comment:
I think that a `recovery` flag (it seems to me, that it is the same as the given reason) can be obtained via `kernal context`, injected into the eviction context. Perhaps, a new method that should be called on recovery phase is option here as well `clearAllOnRecovery(EvictionContext evictionCtx)`

Reply:
Fixed.
Used recovery flag from `kernal context`.